core/meta/inc/TClass.h

```diff
@@ -544,8 +553,16 @@ friend class TProtoClass;
 };
 
 namespace ROOT {
-   template <typename T> TClass* GetClass(      T* /* dummy */)        { return TClass::GetClass(typeid(T)); }
-   template <typename T> TClass* GetClass(const T* /* dummy */)        { return TClass::GetClass(typeid(T)); }
+template <typename T>
```

Comment:
Is this really worth changing? I find the original version much more obvious to read.

Reply:
This was clang-format doing its job...

Reply:
I think that we need to come to a point where clang-format's output *is* the coding standard, or these discussions about coding style will never end. Please adjust the configuration so that it does the right thing, then I can rebase, reformat, and push again.

Reply:
> Please adjust the configuration 

I can't (in good faith) ...  It just got adjusted the other way ...

> we need to come to a point where clang-format's output

As far as I can tell, this is waiting on clang-format to become 'better' (and have more features). i.e. we can not get it to do a good enough job yet.  (on a recent PR, 80% of clang-format's proposal were pessimitation of the layout).